packages/angular_devkit/build_angular/test/browser/bundle-budgets_spec_large.ts

```diff
@@ -64,66 +65,82 @@ describe('Browser Builder bundle budgets', () => {
     await run.stop();
   });
 
-  it(`shows warnings for large component css when using 'anyComponentStyle' when AOT`, async () => {
-    const overrides = {
-      aot: true,
-      optimization: true,
-      budgets: [{ type: 'anyComponentStyle', maximumWarning: '1b' }],
-    };
-
-    const cssContent = `
-      .foo { color: white; padding: 1px; }
-      .buz { color: white; padding: 2px; }
-      .bar { color: white; padding: 3px; }
-    `;
+  cssExtensions.forEach(ext => {
+    it(`shows warnings for large component ${ext} when using 'anyComponentStyle' when AOT`, async () => {
+      const overrides = {
+        aot: true,
+        optimization: true,
+        budgets: [{ type: 'anyComponentStyle', maximumWarning: '1b' }],
```

Comment:
We can add `styles: [`src/styles.${ext}`]` here so that we are sure that the budget only applies to component styles?